api/document/serializers/doc_cursor.py

```diff
@@ -1,13 +1,34 @@
+from typing import Any, Dict, List
+
 from rest_framework import serializers
 
 from document.models import DocNode
 from document.serializers.content import (NestedAnnotationSerializer,
                                           nest_annotations)
 from document.serializers.meta import Meta, MetaSerializer
+from document.tree import DocCursor
+
+JsonDict = Dict[str, Any]
+
+
+class DocCursorField(serializers.Field):
+    def get_attribute(self, instance: DocCursor) -> DocCursor:
+        return instance
+
+
+class ChildrenField(DocCursorField):
+    def to_representation(self, instance: DocCursor) -> List[JsonDict]:
+        return DocCursorSerializer(
+            instance.children(), many=True,
+            context={**self.context, 'is_root': False},
+        ).data
+
+    def to_internal_value(self, data: List[JsonDict]) -> DocCursor:
```

Comment:
I suspect the signature will be `-> List[DocCursor]` at the end of the day, maybe `-> List[DocNode]`? Perhaps we'll do the parent-child relation in `DocCursorSerializer`'s `update()`?

Reply:
Ah cool, I will keep that in mind! Thanks for the feedback.